File: Examples/Curations/Obj-C/CurationsExample/DemoCollectionViewCell.m
//
//  DemoCollectionViewCell.m
//  
//
//  Created by Tim Kelly on 5/25/16.
//
//

#import "DemoCollectionViewCell.h"
#import <SDWebImage/UIImageView+WebCache.h>

@interface DemoCollectionViewCell()

@property IBOutlet UIImageView *feedImageView;

@end

@implementation DemoCollectionViewCell

- (void)setFeedItem:(BVCurationsFeedItem *)feedItem {
    
    super.feedItem = feedItem;
    
    BVCurationsPhoto *curationsPhoto = self.feedItem.photos.firstObject;
    NSURL *imageURL = [NSURL URLWithString:curationsPhoto.url];
    
    [self.feedImageView sd_setImageWithURL:imageURL];
    
}

@end

File: Examples/Curations/Swift/CurationsExample/DemoCollectionViewCell.swift
//
//  DemoCollectionViewCell.swift
//  
//  Copyright © 2016 Bazaarvoice. All rights reserved.
//

import UIKit
import BVSDK
import SDWebImage

class DemoCollectionViewCell: BVCurationsCollectionViewCell {
    
    @IBOutlet weak var feedImageView: UIImageView!
    
    override var feedItem : BVCurationsFeedItem? {
        
        didSet {
            
            let photo : BVCurationsPhoto = self.feedItem!.photos.first!
            let imageUrl : NSURL = NSURL(string:photo.imageServiceUrl)!
            self.feedImageView.sd_setImageWithURL(imageUrl)
            
        }
    }
}
